e2e/pages/organizations/onc-acbs-atls/organization.po.js

```diff
@@ -0,0 +1,86 @@
+const elements = {
+  organizationEditButton: '#edit-organization',
+  organizationName: '#organization-name',
+  organizationWebsite: '#organization-website',
+  saveOrganizationButton: '#chpl-organization-save',
+  organizationList: '.organizations-side-nav',
+  retireOrganization: '#organization-retired',
+  retirementDate: '#retirement-date',
+  addressOnEdit: 'chpl-address',
+  manageUsersPanelHeader: '//*[contains(text(),"Manage Users")]',
+  errorMessage: '.text-danger.ng-scope',
+};
+
+class OrganizationPage {
+  constructor() { }
+
+  organizationNameButton(organizationName) {
+    return $(`//*[contains(text(),"${organizationName}")]`);
+  }
+
+  get organizationList() {
+    return $(elements.organizationList);
+  }
+
+  organizationListCount() {
+    return $(elements.organizationList).length;
```

Comment:
You already define the `organizationList` above, so here it could leverage that definition in such a way that if the locator for the organization list changes, this count wouldn't have to

I think something like this should work:

```suggestion
    return this.organizationList.length;
```